WikipediaUnitTests/Code/FBSnapshotTestCase+WMFConvenience.h

```diff
@@ -21,7 +21,7 @@
  *  @param view The view to verify.
  */
 #define WMFSnapshotVerifyViewForOSAndWritingDirection(view) \
-    FBSnapshotVerifyView((view), [[UIApplication sharedApplication] wmf_systemVersionAndWritingDirection]);
+    FBSnapshotVerifyViewWithOptions((view), [[UIApplication sharedApplication] wmf_systemVersionAndWritingDirection], FBSnapshotTestCaseDefaultSuffixes(), 0.1);
```

Comment:
Was this the lowest tolerance that would work? Seems like 10% of pixels off is a lot of pixels to be off